### ISSUE TITLE
ng new not working

### ISSUE THREAD

**Issue by FructuosoRobles** [angular/angular-cli#3258]
> Please provide us with the following information:
> ---------------------------------------------------------------

### OS?
Mac OSX Sierra

### Versions.
> Please run `ng --version`. If there's nothing outputted, please run in a Terminal: `node --version` and paste the result here:
angular-cli: 1.0.0-beta.21
node: 6.9.1
os: darwin x64

### Repro steps.
> Was this an app that wasn't created using the CLI? What change did you do on your code? etc.
ng new PROJECT_NAME (this command is not working)

### The log given by the failure.
> Normally this include a stack trace and some more information.

not failure in the logs, it creates the files but when I open the directory its empy, no files inside

### Mention any other details that might be useful.

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by deebloo** [comment]
@FructuosoRobles can you elaborate/give the exact steps.

**Comment by wanthony** [comment]
@deebloo I've run into this as well when running `ng new`.

I noticed that instead of creating the app in the current directory, the app is created in my home dir.

My `ng --version`:

```
angular-cli: 1.0.0-beta.21
node: 6.3.1
os: darwin x64
```

**Comment by FructuosoRobles** [comment]
Hello. the steps are:

1.- make dir in desktop: PROJECT
2.- access PROJECT dir from terminal
3.- ng new PROJECT_ANGULAR
4.- all ok!! but, files are created in home dir (true, I just delete them a few minutes ago).


What can I do?

**Comment by wanthony** [comment]
@deebloo @FructuosoRobles I think I tracked down the problem (for me at least, @FructuosoRobles this may apply to you as well) -- seems I had a `package.json` in my home dir, and it looks like `ng new` (well, buried deep in the `Project` model from `ember-cli`, which is eventually used by `ng new`) traverses up the directory tree looking for a `package.json`.  If it's found, it assumes that's the project root, and at some point `ng new` changes to that directory.

Once I removed the rogue `package.json` I was able to run `ng new projectname` and see the generated files in a new directory under my current directory.
